src/components/molecules/HeroMessage/styles.tsx

```diff
@@ -9,7 +9,7 @@ export const StyleHeroMessageWrapper = styled.div `
     flex-direction: column;
     justify-content: space-around;
     max-width: 100%;
-    padding: ${space.m};
+    padding: ${space.s} ${space.m} ${space.m} ${space.m};
```

Comment:
you can use the shorthand -> padding: ${space.s} ${space.m} ${space.m};

Reply:
Oh yeah, I always forget about that. Thanks.